server/channels/api4/plugin_test.go

```diff
@@ -79,7 +83,8 @@ func TestPlugin(t *testing.T) {
 		_, err = client.RemovePlugin(context.Background(), manifest.Id)
 		require.NoError(t, err)
 
-		th.App.Channels().RemovePlugin(manifest.Id)
+		err = th.App.Channels().RemovePlugin(manifest.Id)
+		require.NoError(t, err)
```

Comment:
L83 already removes the plugin. The error is to be expected. I don't know why the test tries to remove the plugin twice. Can you please remove the call to `Channels().RemovePlugin`?

Reply:
Let me try!